migrations/2023-03-16-195722_create_twabs/up.sql

```diff
@@ -0,0 +1,9 @@
+-- Your SQL goes here
+DROP TABLE twabs;
+CREATE TABLE twabs (
+    id SERIAL PRIMARY KEY,
+    "currency address" CHAR(40) NOT NULL,
+    "token address" CHAR(40) NOT NULL,
```

Comment:
I guess we could normalize and have the addresses in other tables? Probably overkill

Reply:
Yeah probably over kill. But I think convention would be "contract_address"

Reply:
Er quote_currency and token_address